data/json/items/containers.json

```diff
@@ -187,6 +187,23 @@
     "symbol": ")",
     "color": "black"
   },
+  {
+    "id": "bag_garbage_reinforced",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "reinforced garbage bag" },
+    "looks_like": "bag_canvas",
+    "description": "Plastic bags placed within each other made to carry much heavier loads, could even carry a dead body inside, it dosent stop the smell though.",
+    "weight": "400 g",
+    "volume": "420 ml",
+    "price": 0,
+    "price_postapoc": 0,
+    "to_hit": -1,
+    "pocket_data": [ { "pocket_type": "CONTAINER", "max_contains_volume": "110 L", "max_contains_weight": "100 kg", "moves": 400 } ],
```

Comment:
Should probably get some penalty in total volume

Reply:
i agree, suggestions? i could just make it a flat 100L